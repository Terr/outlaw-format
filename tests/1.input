=== Begin

ABC

    === DEF
        KEKEKEKE

 === Zomaar een random spatie indent
    === KEK
        
        KEK

        sdf
        sdf
        s
        df
        sd

        sdf


        === Bliep

            V

                === erkeerd

                    LOL

    === KEK II
