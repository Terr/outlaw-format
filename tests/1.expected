=== Begin

    ABC

    * A
    * B

    === DEF

        KEKEKEKE

=== Zomaar een random spatie indent

    === KEK

        KEK

        sdf
        sdf
        s
        df
        sd

        sdf

        * Abcdefghijklmnopqrstu
          vwxyz
        * B
            * C
                * Dddddddd
                  dddd
                  dd

        === Bliep

            V

            === erkeerd

                LOL

    === KEK II
